src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -300,3 +311,23 @@ def get_partition_hosts(self, partition_key, host_list):
             return hosts, partiton_name
         else:
             return host_list, None
+
+    def separate_logs(self, test_name=None):
```

Comment:
Maybe `update_log_file_names()` or `create_log_file_names()` would be more descriptive ... something like that.